### PR TITLE
set monospace as main fonts on demo for nicer looks.

### PR DESCRIPTION
**Pull request by Ar37-rs**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,8 +19,8 @@ include = [
 
 [dependencies]
 egui = "0.13.1"
-egui-miniquad = "0.5.0"
-macroquad = "0.3.6"
```
Comment (optozorax) [inline Cargo.toml:23]: I prefer using exact versions, because macroquad brokes quite often.

File: examples/demo.rs [hunk 2]
```diff
@@ -1,13 +1,14 @@
+use egui::{FontDefinitions, FontFamily, TextStyle, Window as EguiWindow};
+use egui_demo_lib::DemoWindows;
 use macroquad::prelude::*;
 
 fn window_conf() -> Conf {
     Conf {
         window_title: "egui with macroquad".to_owned(),
-        high_dpi: true,
+        high_dpi: false,
```
Comment (optozorax) [inline examples/demo.rs:8]: why?

File: examples/demo.rs [hunk 3]
```diff
@@ -17,26 +18,37 @@ fn draw_primitives() {
 #[macroquad::main(window_conf)]
 async fn main() {
     let mut show_egui_demo_windows = false;
-    let mut egui_demo_windows = egui_demo_lib::DemoWindows::default();
+    let mut egui_demo_windows = DemoWindows::default();
     let mut draw_primitives_after_egui = false;
 
+    // setup (cfg)
+    egui_macroquad::cfg(|ctx| {
```
Comment (optozorax) [inline examples/demo.rs:25]: It's like show an example of how to use new function?

File: examples/demo.rs [hunk 4]
```diff
@@ -49,7 +61,8 @@ async fn main() {
         if draw_primitives_after_egui {
             draw_primitives();
         }
-
-        next_frame().await
+        next_frame().await;
+        // // reduce CPU usage on native (non-web) if need to, put the main thread to sleep for a while after next_frame().
+        // std::thread::sleep(std::time::Duration::from_millis(10));
```
Comment (optozorax) [inline examples/demo.rs:66]: AFAIK this will not work on the web. Maybe remove this comments?

**Review by optozorax — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (Ar37-rs) [reply Cargo.toml:23]: oh i don't know that, if so i agree it's better set the exact versions.

File: examples/demo.rs (hunk 2 above)
Comment (Ar37-rs) [reply examples/demo.rs:8]: i thought it's one of  the causes of high cpu usage, but apparently that's not quite true.

File: examples/demo.rs (hunk 3 above)
Comment (Ar37-rs) [reply examples/demo.rs:25]: yup,

File: examples/demo.rs (hunk 4 above)
Comment (Ar37-rs) [reply examples/demo.rs:66]: yup, native only. but never mind about this.

**Pull request closed without merging**